index.d.ts

```diff
@@ -28,7 +28,7 @@ type MultipartOptions = {
     /**
      * Manage the file stream like you need
      */
-    onFile?: (fieldName: string, stream: Readable, filename: string, encoding: string, mimetype: string) => void;
+    onFile?: (fieldName: string, stream: Readable, filename: string, encoding: string, mimetype: string, body: any) => void;
```

Comment:
Should any be changed to:

```
body: Record<string, BodyEntry>
```

Where `BodyEntry` is modelled on [this](https://github.com/fastify/fastify-multipart/blob/master/index.js#L24-L30), so might be similar to below:

```
export interface BodyEntry {
  data: Buffer,
  filename: string,
  encoding: string,
  mimetype: string,
  limit: false
}
```

Reply:
Thanks, I'll add it 🙂

Reply:
This might be possible to improve, e.g encoding might be of type `TranscodeEncoding` from . [here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/v11/buffer.d.ts#L5)etc.